acados/utils/print.c

```diff
@@ -402,6 +402,39 @@ void print_ocp_qp_out(ocp_qp_out *qp_out)
     return;
 }
 
+
+void print_ocp_qp_out_to_file(FILE *file, ocp_qp_out *qp_out)
```

Comment:
I would move the actual printing routine for an ocp_qp_sol in HPIPM and here just call that one, since the actual QP solution struct is defined in HPIPM and here there is only a typedef.
So the printing routine would remain up to date in case there are changes in HPIPM.